Fix vale warnings in End-user Guide (#2078)
codenvy/che-docs: modules/end-user-guide/partials/proc_defining-maven-repositories-in-the-settings-xml-file.adoc

= Defining repositories in `settings.xml`

To specify your own artifact repositories at `example.server.org`, use the `settings.xml` file.

> To do that, ensure, that `settings.xml` is in all the containers that use Maven tools, in particular the Maven container and the Java plug-in container.

Reviewer: ```suggestion
Ensure that `settings.xml` is in all the containers that use Maven tools. In particular, ensure that it is in the Maven container and the Java plug-in container.
```
